apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/lib/utils.tsx

```diff
@@ -1145,3 +1147,17 @@ export const findHiddenFieldUsedInLogic = (survey: TSurvey, hiddenFieldId: strin
 
   return survey.questions.findIndex((question) => question.logic?.some(isUsedInLogicRule));
 };
+
+export const findEndingCardUsedInLogic = (survey: TSurvey, endingCardId: string): number => {
```

Comment:
I couldn't find any implmentations of this function, pls use this or remove if its not required